src/components/Product.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+
+function Product({typeOfSell, img, title,  typeOfBag, priceOfBag}) {
+
+    return (
+        <article>
+            <span>{typeOfSell}</span>
+            <img src={img} alt={title}/>
+            <p>{typeOfBag}</p>
+            <h4>€{priceOfBag}</h4>
+        </article>
+    )
+}
+
+export default Product
```

Comment:
Consistent overal in je code wel of geen ; achter een zin plaatsen.

Reply:
ja je hebt gelijk